### ISSUE TITLE
Can you add tvos support to the podspec so tvos apps can use it?

### ISSUE THREAD

**Issue by cerupcat** [comyarzaheri/CZWeatherKit#29]
Can you add tvos support to the podspec so tvos apps can use it?

**Comment by comyarzaheri** [comment]
Hi, I've added it to the podspec in [v2.2.2](https://github.com/comyarzaheri/CZWeatherKit/releases/tag/2.2.2) but admittedly didn't test it. Please post another issue if you run into problems, thanks!

**Comment by comyarzaheri** [comment]
Err, seems I jumped the gun a bit...it looks like one of CZWeatherKit's dependencies doesn't support tvOS so this won't work as a quick fix. I'll leave this issue open as I work on figuring out something.

**Comment by cerupcat** [comment]
I sent a pull request to PINCache to add tvos support. They seem response, so we'll see if it gets merged.

I also sent you a pull request, since this won't build without removing the imports for mapkit since that's not supported on tvos.

I've tested and everything appears to be working correctly on tvos.

**Comment by cerupcat** [comment]
Just a note. PINCache podspec was updated to support tvos.

**Comment by comyarzaheri** [comment]
Awesome, thanks for getting that change in for PINCache, I cherry-picked one of your commits from your pull-request into master. I didn't really have time to test the changes in practice, but the unit tests pass. In any case, I didn't want to keep you waiting on me so I went ahead and released [2.2.3](https://github.com/comyarzaheri/CZWeatherKit/releases/tag/2.2.3). Let me know if you run into any issues.
